client/src/pages/Profile.js

```diff
@@ -0,0 +1,224 @@
+import React, { useState } from "react";
```

Comment:
I realize this has been merged already, but I feel like this component could use a bit of refactoring. The css got a little to big and it would be nice to move it to a separate file. I also think the lists could be their own component.